Tests/UnitTests/Purchasing/ConfigurationTests.swift

```diff
@@ -31,50 +31,6 @@ class ConfigurationTests: TestCase {
         expect(Configuration.validate(apiKey: "swRTCezdEzjnJSxdexDNJfcfiFrMXwqZ")) == .legacy
     }
 
-    func testNoObserverModeWithStoreKit1() {
```

Comment:
You brought back `observer_mode_with_storekit2`. Can you bring back these tests instead? I think you can just reset this file, right?